ios/StripeTerminalReactNative.swift

```diff
@@ -961,4 +999,19 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
         let result = Mappers.mapFromReaderDisplayMessage(displayMessage)
         sendEvent(withName: ReactNativeConstants.REQUEST_READER_DISPLAY_MESSAGE.rawValue, body: ["result": result])
     }
+
+    func terminal(_ terminal: Terminal, didChange offlineStatus: OfflineStatus) {
+        let offlineStatus = Mappers.mapFromOfflineStatus(offlineStatus)
+        sendEvent(withName: ReactNativeConstants.CHANGE_OFFLINE_STATUS.rawValue, body: ["result": offlineStatus])
+    }
+    
+    func terminal(_ terminal: Terminal, didForwardPaymentIntent intent: PaymentIntent, error: Error?) {
+        let result = Mappers.mapFromPaymentIntent(intent, uuid: "")
+        sendEvent(withName: ReactNativeConstants.FORWARD_PAYMENT_INTENT.rawValue, body: ["result": result])
```

Comment:
Similar to the Android implementation, we also have to callback with the `error` here.

Reply:
This has been addressed. Resolving comment thread.